client/config.go

```diff
@@ -76,6 +77,9 @@ type Configuration struct {
 	// BackgroundHealthCheckFailThrottleFactor is the factor of the host connect
 	// time to use when sleeping between a failed health check and the next check.
 	BackgroundHealthCheckFailThrottleFactor float64 `yaml:"backgroundHealthCheckFailThrottleFactor" validate:"min=0,max=10"`
+
+	// The configuration for hashing
+	HashingConfiguration HashingConfiguration `yaml:"hashing"`
```

Comment:
For consistency perhaps start the comment with the symbol name?  i.e. `// HashingConfiguration is the configuration for hashing of IDs to vnodes`